packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -150,17 +150,17 @@ class RunCommand extends RunCommandBase {
     };
   }
 
-  Device device;
+  List<Device> devices;
 
   @override
   String get usagePath {
     final String command = shouldUseHotMode() ? 'hotrun' : name;
 
-    if (device == null)
+    if (devices == null)
       return command;
 
     // Return 'run/ios'.
-    return '$command/${getNameForTargetPlatform(device.targetPlatform)}';
+    return '$command/${getNameForTargetPlatform(devices[0].targetPlatform)}';
```

Comment:
shouldn't this be run/all or some such? It seems weird to report the first device's platform in the case of multiple platforms.